pipeline_dp/mean_var_params.py

```diff
@@ -0,0 +1,132 @@
+"""DP computing of count, sum, mean, variance."""
+
+import numpy as np
+import pipeline_dp
+
+from dataclasses import dataclass
+
+
+@dataclass
+class MeanVarParams:
+    """Manages the parameters used for computing the mean/variance."""
+    eps: float
+    delta: float
+    low: float
+    high: float
+    max_partitions_contributed: int
+    max_contributions_per_partition: int
+    noise_kind: pipeline_dp.NoiseKind  # Laplace or Gaussian
+
+    def l0_sensitivity(self):
+        return self.max_partitions_contributed
+
+    def linf_sensitivity(self, metric):
+        return self.max_contributions_per_partition * {
+            pipeline_dp.Metrics.COUNT: 1,
+            pipeline_dp.Metrics.SUM: np.max(self.low, self.high)
```

Comment:
self.low -> abs(self.low), self.high -> abs(self.high)

low and high might be negative, the sensitivity is absolute value of the maximum change of metrics, that's why we need to take abs()